### DISCUSSION TITLE
Greeting, I'd like to make 3D model interactive VR environment.

### DISCUSSION THREAD

**Discussion by webcat359** [community/community#29791] [Other Feature Feedback, Questions, & Ideas] [answered]
Greeting, I'd like to make 3D model interactive VR environment.

**Comment by talented0312** [comment, marked as answer]
To build your project, you can use Blender and Three.js fiber/drei.

I think it can help you.

thanks

**Comment by webcat359** [comment]
thanks for your opinion.

Hope you are doing well!
